pkg/api/events/sorted_event_list.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package kubectl
+package events
```

Comment:
If you split this commit out I will fast-track merge it and reduce the size of this PR by a small amount.